docs/README.md

```diff
@@ -35,37 +40,43 @@ The following steps will guide you to setup GitOps for application delivery:
+
+The bootstrapped environment sets up the necessary pipelines using OpenShift Pipelines required for developers/teams to perform CI with their respective environments, they can then check the progress of their development as CD for the applications is performed using [ArgoCD](https://argoproj.github.io/argo-cd/) constantly synchronising the desired state of the application to the cluster.
+
+Users can easily customize the pipelines to their own needs. A simple push to the GitOps repo can alter the configuration of the GitOps repo and hence the style by which you want to run your organization, by triggering a CI pipeline, further a merge to the trunk branch triggers a deployment that further changes the state of your application on the cluster. Giving teams to use the power of OpenShift/Kubernetes to run powerful applications.
 
-The bootstrapped environment sets up the necessary pipelines using Tekton required for developers/teams to perform CI with their respective environments they are allocated to , they can then check the progress of their development as CD for the applications is performed using [ArgoCD](https://argoproj.github.io/argo-cd/) constantly synchronising the state of the application in the cluster.
+Now we have an a team that set out as a start-up but went on to be a full-fledged organization with multiple verticals and businesses set up in different geographical locations leveraging the use of OpenShift and GitOps with "Git as the source of truth".
 
-Giving users the ability to simply modify the pipelines to their own needs at any point to a more customized approach. A simple push to the GitOps repo can alter the configuration of the GitOps repo and hence the style by which you want to run your organisation, by triggering a CI pipeline , further a merge to master triggers a deployment that further changes the state of your application on the cluster. Giving teams to use the power of OpenShift/Kubernetes to run powerful applications.
+Now's your chance to ramp up your organization using our modernized approach for end-to-end application delivery.
 
-Now we have an a team that set out as a start up but went on to be a full fledged organization with a multiple verticals and business set up in different geographical locations leveraging the use of OpenShift and GitOps with "Git as the source of truth".
+A simple GitOps configuration can be seen [here](https://github.com/rhd-gitops-example/gitops), all achieved with a list of commands listed in [gitops Command Reference](./commands), why wait?
```

Comment:
New location for example gitops repo: https://github.com/redhat-developer/gitops-repo-example